packages/gatsby-source-wordpress/README.md

```diff
@@ -144,6 +144,19 @@ module.exports = {
         normalizer: function({ entities }) {
           return entities
         },
+        // add/remove wordpress data normalizers
+        // each normalizer is an object with name and normalizer property being a function that accepts
+        // object with entities array and different helpers and configuration settings
```

Comment:
After reading this again after so long it occurs to me these comments could be a little clearer. What do you think about this?

```
        // The normalizers option allows you to manipulate the array of internal
        // normalizers that are applied to entities after they're fetched
        // from WordPress.
        // You can add your own normalizers to this array by adding an object
        // that contains name and normalizer properties.
        // Name is the name of your normalizer, and normalizer is a function that
        // should return the array of entities that are passed to it.
        // This is useful if you need more control over the order of normalizers,
        // instead of your normalizer being applied after the built in normalizers (as is the case with the normalizer option).
```

Reply:
Awesome, thanks! Just merging master back into this PR to see if it fixes the  failing tests, then LGTM!